cfgov/v1/tests/test_password_policy.py

```diff
@@ -53,6 +55,11 @@ class TestMinimumPasswordAge(TestWithUser):
 
     def test_too_soon(self):
         user = self.get_user()
+        history_item = user.passwordhistoryitem_set.latest()
+        current_locked_until = history_item.locked_until
+        new_locked_until = current_locked_until.replace(year=datetime.MAXYEAR)
+        history_item.locked_until = new_locked_until
+        history_item.save()
         with self.assertRaises(ValidationError):
```

Comment:
Hmm...  I'll have to think about it some more. It seems like "throws a validation error if locked_until is in the future" is a meaningful test, though. 

Maybe the naming is real flaw.